resources/dicts/events/leader_den/success/other_clan.json

```diff
@@ -1405,11 +1746,33 @@
   },
   {
     "interaction_type": "antagonize",
-    "event_text": "At the Gathering, m_c forgoes a traditional speech in favor of retelling the tale of a legendary o_c_n leader... but with {PRONOUN/m_c/poss} own special graphically violent twist. o_c_n warriors cover their apprentices' ears, and even o_c_n's leader looks like they'll have nightmares.",
+    "event_text": "At the Gathering, o_c_n's leader repeatedly provokes and challenges c_n. m_c remains composed, not a whisker out of place, and asks o_c_n's leader to name a place and time for them to settle the score. {PRONOUN/m_c/poss/CAP} self-possession riles o_c_n's leader even more.",
+    "rel_change": -10,
+    "m_c": {
+      "trait": ["calm"],
+      "skill": ["FIGHTER,1"]
+    },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
+  },
+  {
+    "interaction_type": "antagonize",
+    "event_text": "At the Gathering, m_c snarls and spits at o_c_n's leader. As {PRONOUN/m_c/subject} {VERB/m_c/do/does}, {PRONOUN/m_c/subject} {VERB/m_c/feel/feels} a chill rush over {PRONOUN/m_c/poss} pelt, like plunging into icy water. Starlight glimmers at the paws of o_c_n's leader, but m_c ignores it.",
```

Comment:
hmmm i feel like i don't quite understand the intention of this one?  m_c is being insulting > m_c feels a ghost? > m_c sees ghosty glimmers in the other leader? I'm not really getting how these things are linking together or what is being implied

Reply:
The reason this one is a little sketchy is that I was attempting to work in the Ghost,1 trait -- since it's below both "ghost sight" and "ghost speaker", I wasn't sure how to imply a ghostly presence watching over o_c_n's leader without being more obvious. I edited it to add in a little more information to clarify what I was going for!